src/mainboard/protectli/vault_adl_p/board_info.txt

```diff
@@ -0,0 +1,7 @@
+Category: sbc
+Board URL: TBD
+ROM IC: TBD
```

Comment:
We can fill that in?

Reply:
https://github.com/Dasharo/coreboot/pull/455/commits/aafb2783e1929ecea0a321a2503668102a2b00f2#diff-b81e847a21c839ec7979c93e200e0f9e1a7fb40ba21d0a49a1b2af4e142b6ac9R3